packages/blade/src/utils/useDidUpdate.web.test.ts

```diff
@@ -18,4 +18,18 @@ describe('useDidUpdate', () => {
     // Should be called after first render, anytime dependencies change
     expect(effectCallback).toHaveBeenCalledTimes(1);
   });
+
+  it('should not run the effect when the dependencies do not change', () => {
```

Comment:
can you add ssr test to make sure nothing gets called on server?

Reply:
it's impossible to get a call on server, since useEffect doesn't run there.

Reply:
yes so your test shall have toNotHaveBeenCalled() to true right?

Reply:
Added.